cmd/dfdaemon/app/options/options.go

```diff
@@ -125,6 +128,7 @@ func (o *Options) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&o.DownRule, "rule", "", "download the url by P2P if url matches the specified pattern,format:reg1,reg2,reg3")
 	fs.BoolVar(&o.Notbs, "notbs", true, "not try back source to download if throw exception")
 	fs.StringSliceVar(&o.TrustHosts, "trust-hosts", o.TrustHosts, "list of trusted hosts which dfdaemon forward their requests directly, comma separated.")
+	fs.StringSliceVar(&o.SupernodeList, "super-nodes", []string{"127.0.0.1"}, "specify the addresses(IP:port) of supnernodes that will be passed to dfget.")
```

Comment:
Use `node` instead of `super-nodes`, it's consistent with `dfget`.
It's better not to set dfdaemon's option `super-nodes` a default value. If the this option is not set, then `dfget` use its own nodes.

Reply:
I agree with you. And that has been updated. PATL again.